packages/agreement-process/src/services/agreementStampUtils.ts

```diff
@@ -27,7 +28,7 @@ export const createStamp = (
     : undefined;
 
   return {
-    who: authData.userId,
+    who: authData?.userId ?? unsafeBrandId("platform"),
```

Comment:
we should remember to check (and not add in the future) something that validates if these ids are valid uuid otherwise we could have a problem with this "platform" placeholder.

Is it used in other parts of the codebase this approach?

Reply:
I think in this case we shall not set the stamp, for two reasons:
1. Stamps are needed to audit operations performed by users - in this case, it's not a user operation but an automation from the platform, it's sufficient to have a dedicated event with a clear name
2. When an agreement is suspendedByPlatform, we don't set any stamp and there is no `suspensionByPlatform` stamp (see [here](https://github.com/pagopa/interop-be-monorepo/blob/ce5f012840fc95a979d603b0831ab9b7cb5454bb/packages/agreement-process/src/services/agreementStateProcessor.ts#L251-L252) and [here](https://github.com/pagopa/interop-be-monorepo/blob/ce5f012840fc95a979d603b0831ab9b7cb5454bb/packages/agreement-process/src/services/agreementSuspensionProcessor.ts#L78-L79))